crates/context_aware_config/src/api/dimension/handlers.rs

```diff
@@ -208,3 +214,33 @@ async fn delete_dimension(
         ))
     }
 }
+
+#[put("/position/update")]
+async fn temp_position_update(
+    user: User,
+    db_conn: DbConnection,
+) -> superposition::Result<HttpResponse> {
+    let DbConnection(mut conn) = db_conn;
+    let results: Vec<(String, i32, i32)> = dimensions
+        .order(priority.asc())
+        .select((dimension, priority, position))
+        .load::<(String, i32, i32)>(&mut conn)
+        .map_err(|err| {
+            log::error!("failed to fetch dimensions with error: {}", err);
+            unexpected_error!("Something went wrong")
+        })?;
+
+    let _ = for (index, (dim, pri, _)) in results.iter().enumerate() {
+        diesel::update(dimensions)
+            .filter(dimension.eq(dim))
+            .filter(priority.eq(*pri))
```

Comment:
why do we need to check the priority value over here ?
is checking for dimension name not enough ?